level2/backend-l2.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  backend "s3" {
+    bucket         = data.aws_s3_bucket.example_bucket.id 
+    key            = "fsb.tfstate"
```

Comment:
The key name should be different from the level1 key name also you should specify the same bucket name as level1